Use macro since SPIN_LOCK_UNLOCKED has been obsolete in kernel since 2007

bicatali@gentoo.org 07/2012

--- cvmfs-2.0.19.orig/kernel/cvmfsflt/src/cvmfsflt_data.c
+++ cvmfs-2.0.19/kernel/cvmfsflt/src/cvmfsflt_data.c
@@ -5,7 +5,7 @@
 int rbuf_head, rbuf_tail; /* Current positions in ring buffer */
 struct semaphore rbuf_full, rbuf_empty;   /* semaphores for producer-consumer synchronization */
 struct cvmfsflt_logline *ll_current = NULL;
-static spinlock_t rbuf_wlock = SPIN_LOCK_UNLOCKED;
+DEFINE_SPINLOCK(rbuf_wlock);
 atomic_t cvmfsflt_noll = ATOMIC_INIT(0);  /* number of loglines in the ring buffer */
 
 
